rust/sbp/src/messages/imu.rs

```diff
@@ -176,6 +179,19 @@ impl super::SBPMessage for MsgImuRaw {
     fn write_frame(&self, frame: &mut Vec<u8>) -> std::result::Result<(), crate::FramerError> {
         crate::write_frame(self, frame)
     }
+
+    #[cfg(feature = "swiftnav-rs")]
+    fn gps_time(
+        &self,
+    ) -> Option<std::result::Result<crate::time::MessageTime, crate::time::GpsTimeError>> {
+        let tow_s = (self.tow as f64) / 1000.0;
```

Comment:
Is is correct to assume that the tow field always is in milliseconds?

Reply:
@jbangelo @denniszollo are there some messages that report tow in seconds, not ms?

@notoriaga there is a "units" field in the yaml spec that we could potentially inspect here.

Reply:
I added an assert that the units are ms. Only one wasn't `IMU_MSG_RAW`. For now just skipping that msg, it seems like it might not actually be representing 'rover' time - but rather 'Time since reference epoch in milliseconds' which seems like not what we want